games/nasb2/base_files/README.md

```diff
@@ -1,10 +1,6 @@
 # Nickelodeon All-Star Brawl 2
 
-## Description: 
+## Description: Last updated: July 16th 2024
 
-NASB Sweep!!!
-
-## Credits: 
-
-TNTBlowzUp
+## Credits: Renders made by Fairplay Labs, implementation by TNTBlowzUp
```

Comment:
FYI edits made in the MD file will be overwritten by the data from the `config.json`